doc/COOKBOOK-FIXERS.md

```diff
@@ -128,7 +128,7 @@ final class RemoveCommentsFixerTest extends AbstractFixerTestCase
 
     public function provideFixCases()
     {
-        return array();
+        return [];
```

Comment:
What is actually 2.2 for if there was 2.3  released already? And why 2.1 is not?
I'm kinda confused by that.